merge_rgba/__init__.py

```diff
@@ -5,11 +5,16 @@
 from rasterio.transform import Affine, rowcol
 
 
-logger = logging.getLogger('merge_rgba')
+logger = logging.getLogger(
+        "merge_rgba"
+        
+        
+        )
```

Comment:
I installed the `pre-commit` hook which caught and fixed the above change locally before the commit was pushed. 👍 

Then I removed the pre-commit hook, tried again and pushed the change - hoping that the `pre-commit` call in the Travis tests would catch it and exit with an error. Unfortunately, it looks like the tests passed anyways and the change snuck in anyhow. Not sure why, I need to look at the pre-commit cli more closely.